lib/vsc/administration/slurm/sync.py

```diff
@@ -116,11 +116,12 @@ def slurm_project_qos(projects, slurm_qos_info, clusters, protected_qos, qos_cle
             qos_name = f"{cluster}-{project.name}"
             if qos_name not in cluster_qos_names:
                 commands.append(create_add_qos_command(qos_name))
-            commands.append(create_modify_qos_command(qos_name, {
-                "GRPTRESMins": "billing={credits},gres/gpu={gpuminutes}".format(
-                    credits=60*int(project.credits),
-                    gpuminutes=max(1, 60*int(project.gpu_hours)))
-                }))
+            commands.append(
+                create_modify_qos_command(
+                    qos_name,
+                    {f"GRPTRESMins": f"billing={60*int(project.credits)},gres/gpu={max(1, 60*int(project.gpu_hours))}"},
```

Comment:
line too long? why was this splitted?

Reply:
blame the unit tests in `vsc-install`, imo the code here was perfectly fine as it was